engine/src/main/java/org/entando/entando/aps/servlet/AuthEndpointServlet.java

```diff
@@ -130,7 +130,7 @@ private boolean validateClient(final OAuthAuthzRequest oauthRequest, HttpServlet
                     throw OAuthUtils.handleOAuthProblemException("Invalid clientId");
                 } else if (clientDetail.getExpirationDate().getTime() < System.currentTimeMillis()) {
                     throw OAuthUtils.handleOAuthProblemException("ClientId is expired");
-                } else if (!clientDetail.getCallbackUrl().equals(oauthRequest.getRedirectURI())) {
+                } else if (!oauthRequest.getRedirectURI().endsWith(clientDetail.getCallbackUrl())) {
```

Comment:
This needs to be changed back, after we decide in the best approach to add the full URL in DB, see my other comment

Reply:
look at the previous note